api/src/services/fields.ts

```diff
@@ -399,7 +399,10 @@ export class FieldsService {
 			if (hookAdjustedField.schema) {
 				const existingColumn = await this.schemaInspector.columnInfo(collection, hookAdjustedField.field);
 
-				if (!isEqual(sanitizeColumn(existingColumn), hookAdjustedField.schema)) {
+				// Sanitize column only when applying snapshot diff
+				const columnToCompare = opts?.bypassLimits ? sanitizeColumn(existingColumn) : existingColumn;
```

Comment:
Only a small question: How safe is it to bet on this option? I think it's okay, but I wonder if we should pass another option à la `isSnapshotChange` when applying snapshot diffs to be able to recognize for sure 🤔

Reply:
As discussed on Discord we can keep it like this 👍